dmq-node/src/DMQ/Configuration/CLIOptions.hs

```diff
@@ -53,6 +54,20 @@ parseCLIOptions =
           <> help "Topology file for DMQ Node"
           )
         )
+    <*> optional (
+          strOption
+          (  long "cardano-node-socket"
+          <> metavar "Cardano node socket path"
+          <> help "Used for local connections to Cardano node"
+          )
+        )
+    <*> optional (
+          option auto
+          (  long "cardano-network-magic"
+          <> metavar "Cardano node network magic"
+          <> help "The network magic of cardano-node client for local connections"
```

Comment:
We should find a way to show the defaults values that are in `Configuration` module.  I'll create an issue for it.